src/react-native/javascript.ts

```diff
@@ -101,3 +99,229 @@ Sentry.init({
   // spotlight: __DEV__,
 });`;
 }
+
+function getMainAppFilePath(): string | undefined {
+  const prefixGlob = '{.,./src,./app}';
+  const suffixGlob = '@(j|t|cj|mj)s?(x)';
+  const universalGlob = `@(App|_layout).${suffixGlob}`;
+  const jsFileGlob = `${prefixGlob}/+(${universalGlob})`;
+  const jsPath = traceStep('find-app-js-file', () =>
+    getFirstMatchedPath(jsFileGlob),
+  );
+  return jsPath;
+}
+
+/**
+ * This step should be executed after `addSentryInit`
+ */
+export async function wrapRootComponent() {
+  const showInstructions = () =>
+    showCopyPasteInstructions(
+      'App.js or _layout.tsx',
+      getSentryWrapColoredCodeSnippet(),
+    );
+
+  const jsPath = getMainAppFilePath();
+  Sentry.setTag('app-js-file-status', jsPath ? 'found' : 'not-found');
+  if (!jsPath) {
+    clack.log.warn(
+      `Could not find main App file. Please wrap your App's Root component.`,
+    );
+    await showInstructions();
+    return;
+  }
+
+  const jsRelativePath = path.relative(process.cwd(), jsPath);
+
+  const js = fs.readFileSync(jsPath, 'utf-8');
+
+  const mod = parseModule(js);
+  const result = checkAndWrapRootComponent(mod);
+
+  if (result === SentryWrapResult.AlreadyWrapped) {
+    Sentry.setTag('app-js-file-status', 'already-includes-sentry-wrap');
+    clack.log.warn(
+      `${chalk.cyan(
+        jsRelativePath,
+      )} already includes Sentry.wrap. We wont't add it again.`,
+    );
+    return;
+  }
+
+  if (result === SentryWrapResult.NotFound) {
+    clack.log.warn(
+      `Could not find your App's Root component. Please wrap your App's Root component manually.`,
+    );
+    await showInstructions();
+    return;
+  }
+
+  traceStep('add-sentry-wrap', () => {
+    clack.log.success(
+      `Added ${chalk.cyan('Sentry.wrap')} to ${chalk.cyan(jsRelativePath)}.`,
+    );
+
+    fs.writeFileSync(jsPath, generateCode(mod.$ast).code, 'utf-8');
+  });
+
+  Sentry.setTag('app-js-file-status', 'added-sentry-wrap');
+  clack.log.success(
+    chalk.green(`${chalk.cyan(jsRelativePath)} changes saved.`),
+  );
+}
+
+export enum SentryWrapResult {
+  NotFound = 'RootComponentNotFound',
+  AlreadyWrapped = 'AlreadyWrapped',
+  Success = 'Success',
+}
+
+export function checkAndWrapRootComponent(
+  mod: ProxifiedModule,
+): SentryWrapResult {
+  if (doesContainSentryWrap(mod.$ast as t.Program)) {
+    return SentryWrapResult.AlreadyWrapped;
+  }
+
+  const defaultExport = getDefaultExport(mod.$ast as t.Program);
+  if (defaultExport === undefined) {
```

Comment:
This could be just -> 
````suggestion
  if (!defaultExport) {
````

Reply:
Yep :)